content/millennium/r4/foundation/conformance/structure-definition.md

```diff
@@ -147,6 +148,7 @@ Authorization is not required.
 [`financial-transaction-tendered-amount`]: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-tendered-amount?_format=json
 [`financial-transaction-type`]: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-type?_format=json
 [`formatted-text`]: https://fhir-ehr.cerner.com/r4/StructureDefinition/formatted-text?_format=json
+[`is-modifiable`]: https://fhir-ehr.cerner.com/r4/StructureDefinition/modifiable?_format=json
```

Comment:
is this the right url?

Reply:
https://github.com/cerner/fhir.cerner.com/pull/697/commits/bc9407c18cb420c2ad537b1a0c384d77f3647634#diff-6d3680a1fcdfd40913793f8a19ab5e89c212edcba4a5b88a7ba7cff44c3af047R151